ruma-client-api/src/r0/membership/joined_members.rs

```diff
@@ -80,13 +80,13 @@ mod test {
         assert_matches!(
             from_json_value::<RoomMember>(json!({
                 "display_name": "alice",
-                "avatar_url": "mxc://localhost:wefuiwegh8742w",
+                "avatar_url": "mxc://localhost/wefuiwegh8742w",
             })).unwrap(),
             RoomMember {
                 display_name: Some(display_name),
                 avatar_url: Some(avatar_url),
             } if display_name == "alice"
-                && avatar_url == "mxc://localhost:wefuiwegh8742w"
+                && &avatar_url.to_string() == "mxc://localhost/wefuiwegh8742w"
```

Comment:
Is the extra `&` needed here?

Reply:
We can also use `String::from()` for the left side of the comparison.

Reply:
I would have expected `(foo: Sting) == "string constant"` to compile. If it doesn't, let's keep the code as-is.

Reply:
actually it does compile without it.